### DISCUSSION TITLE
linux小主机真的无解吗

### DISCUSSION THREAD

**Discussion by kndxhz** [ZiHaoSaMa66/OsEasy-ToolBox#46] [General]
现在好多学校机房都使用噢易的linux小主机贴在显示器背面，
然后远程学校服务器qemu的Windows，
虽说kill了Windows的进程可以避免监控和远程关机等操作，
但是广播时是直接广播到linux小主机的，
然后小主机直接给显示器输出，
学校服务器上的Windows根本不参与广播，
并且键鼠操作直接全部从linux小主机屏蔽，好像彻底无解了
（下个星期去看看版本号多少）

另外特别感谢大佬的作品！

**Comment by kndxhz** [comment]
看过教师机 的视角，似乎教师机的软件和你的演示视频中的ui完全不一样

**Comment by ZiHaoSaMa66** [comment]
我嘞个小主机（捂脸
看你的描述 如果这个小主机没操作空间的话 
那可能确实是无解了

**Comment by ZiHaoSaMa66** [comment]
如果可以的话 拷一份学生端的备份过来看看有没有操作空间
没有什么比较傻子的漏洞的话
那可能没办法了（捂脸

**Comment by kndxhz** [comment]
最起码我自己观察了下，开机直接进Linux加载界面，然后进选择服务器界面，最后远程Windows，或许真的除了密码没有其他方式了吧



---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 日期 | 2024年10月02日 15:29 |
| 收件人 | ***@***.***> |
| 抄送至 | ***@***.***>***@***.***> |
| 主题 | Re: [ZiHaoSaMa66/OsEasy-ToolBox] linux小主机真的无解吗 (Issue #25) |

如果可以的话 拷一份学生端的备份过来看看有没有操作空间
没有什么比较傻子的漏洞的话
那可能没办法了（捂脸

**Comment by kndxhz** [comment]
我进不了他的Linux，大概率拷不了，Windows的学生端也没啥用



---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 日期 | 2024年10月02日 15:29 |
| 收件人 | ***@***.***> |
| 抄送至 | ***@***.***>***@***.***> |
| 主题 | Re: [ZiHaoSaMa66/OsEasy-ToolBox] linux小主机真的无解吗 (Issue #25) |

如果可以的话 拷一份学生端的备份过来看看有没有操作空间
没有什么比较傻子的漏洞的话
那可能没办法了（捂脸

**Comment by kndxhz** [comment]
等我什么时候有机会了看看能不能搞到，最起码现在看来这种直接投屏给显示器的根本无解

**Comment by kndxhz** [comment]
另外我把您的软件内嵌到我的软件中应该不建议吧（懒得重新写了）
https://github.com/kndxhz/Let-s-happy

**Comment by ZiHaoSaMa66** [comment]
👌👌

**Comment by kndxhz** [comment]
有解，但是在Windows里操作不了，要在linux里操作，具体方法需要吗（不需要就不发了，有可能会被修复）

**Comment by Tanghaolin0508** [comment]
请问具体的方法是怎么操作的?我学校的噢易就是用Linux小主机。

**Comment by kndxhz** [comment]
> 请问具体的方法是怎么操作的?我学校的噢易就是用Linux小主机。

请确保你有linux基础我才能教，要不然轻易搞费一台主机

**Comment by Tanghaolin0508** [comment]
我学过Linux，请问怎么操作？

**Comment by kndxhz** [comment]
> 我学过Linux，请问怎么操作？

正常进入Windows，用这个仓库的工具杀掉Windows的客户端（客户端默认密码oseasy），鼠标划到屏幕上方中间调出功能栏，点一下功能栏空白处，按ctrl+alt+u呼出linux终端，随后
sudo mv /os-easy.mmc /os-easy.mm
exit
在下课前几分钟输入
sudo mv /os-easy.mm /os-easy.mmc
防止影响其他班的学生

如果已经被广播，就拔掉网线，然后输入上方命令后再插上
**注意：不要瞎搞**

**Comment by kndxhz** [comment]
也不知道会不会被这个公司的程序员看到然后修复，虽然不太可能更新，这大概率是调试用的后门
ctrl+alt+t显示设置
ctrl+alt+p音频设置

**Comment by ZiHaoSaMa66** [comment]
酷

**Comment by Tanghaolin0508** [comment]
> > 我学过Linux，请问怎么操作？
> 
> 正常进入Windows，用这个仓库的工具杀掉Windows的客户端（客户端默认密码oseasy），鼠标划到屏幕上方中间调出功能栏，点一下功能栏空白处，按ctrl+alt+u呼出linux终端，随后
> sudo mv /os-easy.mmc /os-easy.mm
> exit
> 在下课前几分钟输入
> sudo mv /os-easy.mm /os-easy.mmc
> 防止影响其他班的学生
> 
> 如果已经被广播，就拔掉网线，然后输入上方命令后再插上
> **注意：不要瞎搞**

收到，谢谢！

**Comment by kndxhz** [comment]
针对于linux小主机的工具箱
https://github.com/kndxhz/OsEasy_Cloud_ToolBox
大部分功能是重复的，但是我做了一个适用于小主机的解网络
